cookbook-react/src/recipes/buttons/zecat/styles.module.scss

```diff
@@ -0,0 +1,41 @@
+$denim: #0B43D1;
+$smalt: #052E91;
+$white: #FFF;
+$black: #000;
+$mercury: #E1E1E1;
+
+.button {
+  border-radius: 5px;
+  color: $white;
+  font-size: 16px;
+  min-width: 200px; // Just for presentational purposes, remove in real project
```

Comment:
Maybe we can move this to a different css file with a comment to be deleted, and apply its style to the container div

Reply:
I don't know, I don't think this is a problem. Each project will remove this, or else they'll have fixed button sizes.